rust/src/io/reader.rs

```diff
@@ -352,6 +354,16 @@ async fn _read_fixed_stride_array(
     .await
 }
 
+async fn read_null_array(
```

Comment:
Can this function be a regular / non-async function?

Reply:
It can, just wanted to keep it consistent with the other `read_*_array` methods. Should change to non-async?

Reply:
if possible, lets just use the `non-async` function for its simplicity.

Reply:
Done